Header header

int16 accelerometer1
int16 accelerometer2
int16 accelerometer3
int16 magnetometer1
int16 magnetometer2
int16 magnetometer3
int16 gyro1
int16 gyro2
int16 gyro3
